.github/workflows/ant.yml

```diff
@@ -0,0 +1,45 @@
+name: Make self-contained Windows EXE
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+    - v*
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Java runtime
+      uses: actions/setup-java@v3
+      with:
+        distribution: zulu
```

Comment:
Is there a specific reason for using Zulu over for example Temurin?

Reply:
Yes, Temurin didn't have Java 17, at least not for `jdk+fx`.